rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -1,5 +1,8 @@
+local json = require("cjson")
+
 -- this is the Lua representation of Configuration struct in internal/ingress/types.go
 local configuration_data = ngx.shared.configuration_data
+local certificate_data = ngx.shared.certificate_data
```

Comment:
you need to declare this dictionary in nginx.tmpl. Also what size do you think of configuring for it? Given you configure it to be 60MB, how many certificates can we fit?

Reply:
Assuming the certificate is generated with a 2048 bit RSA key/cert pair, the PEM encoded certificate and private key would ~3.2kB. So if we have a dictionary of 60mB we would be able to store 18 750 certificates. With that, I think setting it to 16MB would be good enough (5000 certificates). What do you think?

Reply:
Later we should add this info into docs.